chevah_build

```diff
@@ -89,10 +86,15 @@ PIP_LIBRARIES_OPENSSL_098="\
     pyOpenSSL==16.0.0 \
     scandir==${SCANDIR_VERSION} \
     "
+# Python modules installed through pip on systems not build around cffi.
```

Comment:
Maybe use this comment.

Python modules resulting in OS/CPU specific binaries, but which don't depend on cffi.

---------

The idea is that we have cffi working, with the exception of cryptography, all the other python modules no longer create OS/CPU specific binaries.

Reply:
I don't know… We don't split them on this criteria for other OS'es.